lua/mason-registry/textlint/init.lua

```diff
@@ -0,0 +1,11 @@
+local Pkg = require "mason-core.package"
+local npm = require "mason-core.managers.npm"
+
+return Pkg.new {
+    name = "textlint",
+    desc = [[The pluggable natural language linter for text and markdown.]],
+    homepage = "https://textlint.github.io",
+    languages = {},
+    categories = { Pkg.Cat.Linter },
+    install = npm.packages { "textlint", bin = { "textlint" } },
+}
```

Comment:
NB: Textlint seems to take have a very plugin-centric approach, meaning if you want to use plugins with textlint you'll have to install them in the same location. This is something that Mason doesn't support nicely at the moment.